integration_tests/sdk/aqueduct_tests/conftest.py

```diff
@@ -31,3 +35,9 @@ def test_relational_data_integrations_only(data_integration):
                 "Skipped for data integration `%s`, since it is not of type `%s`."
                 % (data_integration.name(), ",".join(enabled_data_integration_types))
             )
+
+
+def pytest_sessionfinish(session, exitstatus):
```

Comment:
@kenxu95 could you confirm if these changes make sense?

Reply:
Not sure what the `hasattr(session.config, "workerinput")` is about, can you add a comment explaining?

Reply:
Do you know if this is only ever called once, even when the tests are run with concurrency?

Reply:
No this calls when and after every worker ends, the exact use of the line is to only run after all workers ends

Reply:
Can you add that as a comment here? Since it's not intuitive

Reply:
Bump on adding a comment here about the `hasattr(..., "workflowinput")` thingy